proposals/1/standardized-mod-format.md

```diff
@@ -0,0 +1,126 @@
+# Standardized Mod Format (DRAFT!)
+
+## Description Of Issue
+
+Lack of a clear specification between modloaders on a common supported format.
+
+## Solution
+
+A specification which exactly defines specific terms, and does not define legacy formats from the previous era.
+
+### Advantages
+
+- Modloaders and modders know what is supported everywhere.
+
+- Less likelihood to use outdated mechanisms from the old era
+
+- APIs defined in other specifications may gain reference implementations using this specification - these can then become the only implementation of those APIs, preventing fragmentation.
+
+### Disadvantages
+
+- Per-modloader extensions not covered, theoretically limiting features for more portable mods.
```

Comment:
The spec needs to include the handling of features unique to certain modloaders.